src/jquery.dataTables.yadcf.js

```diff
@@ -1884,67 +1884,139 @@ if (!Object.entries) {
 			columnObj = getOptions(oTable.selector)[column_number];
 
 			//add a wrapper to hold both filter and reset button
-			$(filter_selector_string).append("<div onmousedown=\"yadcf.stopPropagation(event);\" onclick=\"yadcf.stopPropagation(event);\"  id=\"" + filter_wrapper_id + "\" class=\"yadcf-filter-wrapper " + columnObj.style_class + "\"></div>");
+			$(filter_selector_string).append($("<div>", {
+				onmousedown: "yadcf.stopPropagation(event);",
+				onclick: "yadcf.stopPropagation(event);",
+				id: filter_wrapper_id,
+				class: "yadcf-filter-wrapper " + columnObj.style_class
+			}));
 			filter_selector_string += " div.yadcf-filter-wrapper";
 			filter_selector_string_tmp = filter_selector_string;
 
 
-			$(filter_selector_string).append("<div id=\"yadcf-filter-wrapper-inner-" + table_selector_jq_friendly + "-" + column_number + "\" class=\"yadcf-filter-wrapper-inner" + " -" + table_selector_jq_friendly + "-" + column_number + "\"></div>");
+			$(filter_selector_string).append($("<div>", {
+				id: "yadcf-filter-wrapper-inner-" + table_selector_jq_friendly + "-" + column_number,
+				class: "yadcf-filter-wrapper-inner" + " -" + table_selector_jq_friendly + "-" + column_number
+			}));
```

Comment:
Here, and in a couple of other places, one of the class names begins with `-`. Perhaps the separating space shouldn't be there. I kept the original code.